interface/usergroup/user_info_ajax.php

```diff
@@ -45,8 +36,8 @@
 $errMsg='';
 $success=update_password($_SESSION['authId'], $_SESSION['authId'], $curPass, $newPass, $errMsg);
 if ($success) {
-    echo xlt("Password change successful");
+    echo "<div class='alert alert-success'>" . xlt("Password change successful") . "</div>";
```

Comment:
@zbig01 , Note I also made a single line change in your flat red css here, check out above link (where it skips whitespace changes; most of us have editors set up to remove empty trailing spacing which there was quite of bit of in your flat red theme).